docs/Assets.md

```diff
@@ -56,44 +65,5 @@ Normally applications are expected to import only the individual icons that are
 
 `import "@ui5/webcomponents-icons/dist/add.js`";`
 
-However, sometimes it makes sense to import all icons, hence the `import "@ui5/webcomponents-fiori/dist/Assets.js";` JSON import. 
+However, sometimes it makes sense to import all icons, hence the `import "@ui5/webcomponents-fiori/dist/Assets.js";` JSON import.
 Along with the icons, it also includes all translatable texts.
-
-<a name="bundling"></a>
-## Efficient asset bundling
-
-You may notice that `Assets.js` imports, such as:
-
-`import "@ui5/webcomponents/dist/Assets.js"`
- 
- will produce warning messages in the browser's console, such as for example:
-> Inefficient bundling detected: consider bundling i18n/theme properties imports as URLs instead of inlining them.
-> See rollup-plugin-url or webpack file-loader for more information.
-> Suggested pattern: "assets\/.*\.json"
-
-What this means is that it's recommended to instruct your source code bundling software
-(some of the most popular being Webpack and Rollup) not to include all the asset files or theming related files
-(files that match the <code>assets\/.*\.json</code> pattern) in your applications' JavaScript bundle,
-but rather to leave them out. At runtime, they will be fetched on demand, if ever requested.
-
-[How to do it with Webpack](https://github.com/webpack-contrib/file-loader)
-
-[How to do it with Rollup](https://github.com/rollup/rollup-plugin-url)
-
-Rollup example:
-
-```js
-import url from "rollup-plugin-url";
-...
-plugins.push(url({
```

Comment:
IMO we should still leave this, but for the json() plugin rather than the url() plugin. We should explain that for webpack it's native, for rollup with a plugin + example and for the newer ones native. Maybe even a table for rollup, webpack, snowpack,etc....?